internal/driver/commands.go

```diff
@@ -59,14 +60,6 @@ func (c *command) help(name string) string {
 	return message + "\n"
 }
 
-// AddCommand adds an additional command to the set of commands
-// accepted by pprof. This enables extensions to add new commands for
-// specialized visualization formats. If the command specified already
-// exists, it is overwritten.
-func AddCommand(cmd string, format int, post PostProcessor, desc, usage string) {
```

Comment:
I added this back in. Note that the function signature is strange since it makes no distinction between a data post-processor and a visualizer.

I looked at the various uses cases of custom commands, and it seems to me that many of them have a distinct separation of data-processing vs visualization (or output storage).